astra/src/main/java/com/slack/astra/chunk/ReadOnlyChunkImpl.java

```diff
@@ -341,7 +341,9 @@ private void cacheNodeListener(CacheSlotMetadata cacheSlotMetadata) {
           Thread.ofVirtual().start(() -> handleChunkAssignment(cacheSlotMetadata));
         } else if (newSlotState.equals(Metadata.CacheSlotMetadata.CacheSlotState.EVICT)) {
           LOG.info("Chunk - EVICT received - {}", cacheSlotMetadata);
-          if (!cacheSlotLastKnownState.equals(Metadata.CacheSlotMetadata.CacheSlotState.LIVE)) {
+          if (!cacheSlotLastKnownState.equals(Metadata.CacheSlotMetadata.CacheSlotState.LIVE)
+              && !cacheSlotLastKnownState.equals(
+                  Metadata.CacheSlotMetadata.CacheSlotState.LOADING)) {
```

Comment:
could be something like this instead for legibility
```suggestion
          if (!
          EnumSet.of(Metadata.CacheSlotMetadata.CacheSlotState.LIVE, Metadata.CacheSlotMetadata.CacheSlotState.LOADING).contains(cacheSlotLastKnownState)) {
```

Reply:
i'll try. its partially weird because of the formatter running. but it might look better like this.